charts/catalog/templates/apiserver-deployment.yaml

```diff
@@ -39,7 +39,7 @@ spec:
         - {{ .Values.apiserver.audit.logPath }}
         {{- end}}
         - --enable-admission-plugins
-        - "KubernetesNamespaceLifecycle,DefaultServicePlan,ServiceBindingsLifecycle,ServicePlanChangeValidator,BrokerAuthSarCheck"
+        - "NamespaceLifecycle,DefaultServicePlan,ServiceBindingsLifecycle,ServicePlanChangeValidator,BrokerAuthSarCheck"
```

Comment:
This now refers to a `k8s.io/apiserver` version (see https://github.com/kubernetes/apiserver/blob/master/pkg/admission/plugin/namespace/lifecycle/admission.go), which gets automatically registered in https://github.com/kubernetes/apiserver/blob/master/pkg/server/plugins.go#L30

Reply:
To double check, I simply made this change against the current release of Service Catalog running in a cluster, and it worked with no issues.